src/test/java/org/jabref/logic/bibtex/FieldWriterTests.java

```diff
@@ -1,26 +1,25 @@
 package org.jabref.logic.bibtex;
 
-import java.util.Collections;
+import java.util.List;
 
 import org.jabref.logic.util.OS;
 import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.field.UnknownField;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.Answers;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.mock;
 
 class FieldWriterTests {
 
     private FieldWriter writer;
 
     @BeforeEach
     void setUp() {
-        this.writer = new FieldWriter(mock(FieldWriterPreferences.class, Answers.RETURNS_DEEP_STUBS));
+        FieldWriterPreferences fieldWriterPreferences = new FieldWriterPreferences(false, List.of(StandardField.MONTH), new FieldContentFormatterPreferences());
```

Comment:
@koppor  Here is the test for the FieldWriter using the FieldWriterPreferenes with Field Month

Reply:
Yeah - and I think, we don't have a test for the content written down at https://github.com/JabRef/jabref/issues/7010. Thus, I am thinking of how to include a test for that. Think, I have a bit of a clue. However, I need the comments of https://github.com/JabRef/jabref/pull/8404 to really know where to put the tests to 😋

Reply:
I will just add one for comment field in this class